docs/sources/developer/architecture/components/metas.md

```diff
@@ -0,0 +1,105 @@
+initialization and it is the responsability of the end-user to define it.
+
+Properties:
+
+- `name` - the name of the app
+- `version` - the version of the app
+- `release` - a release identifier for the app, like a commit hash, a build number or a Docker container tag
+- `environment` - the environment where the app is running, like `staging` or `production`
+
+### Browser
+
+The `browser` meta helps with identifying the environment where the app is running and it should not change across a
+session. Altough it is not handled automatically by the core package, nor it is the responsability of the end-user to
+define it. Wrapper packages like `web-sdk` should handle it.
+
+Properties:
+
+- `name` - the name of the browser
+- `version` - the version of the browser
+- `os` - the operating system name and version where the browser is running
+- `mobile` - a flag indication if the browser is running on a mobile device
+
+### Page
+
+The `page` meta helps developers identify the page where a specific signal is coming from. It is changing automatically
+across a session and even though it is not handled automatically by the core package, wrapper packages like `web-sdk`
+should handle it. But unlike other metas, it can be also overwritten by the end-user if they have a custom mechanism for
+tracking the current page.
+
+Properties:
+
+- `url` - the URL of the current page
+- `id` - the name of the browser
+- `attributes` - a key-value object with additional attributes about the current page
+
+### SDK
+
+The `sdk` meta defines the following properties about the Faro library itself:
+
+- `name` - the name of the core library
+- `version` - the version of the library
+- `integrations` - the list of instrumentations that are used, identified with by the name and the version
+
+The `sdk` meta is handled internally by the core package and the end-user should not change it.
+
+### Session
+
+The `session` meta is a static meta that is used to link signals between them. It is not handled automatically by the
+core package and wrapper packages like `web-sdk` can handle it automatically. But unlike other metas, it can be also
+overwritten by the end-user if they have a different way of defining what a session in.
+
+Properties
+
+- `id` - the name of the browser
+- `started` - the date when the session started
+- `attributes` - a key-value object with additional attributes about the session page
+
+### User
+
+The `user` meta ties signals with a specific user. It is not required but it can be provided during initialization as
+well as programatically once the Faro library was initialized.
+
+Properties:
+
+- `id` - the ID of the user
+- `username` - the username of the user
+- `email` - the user's email
+- `attributes` - a key-value object with additional attributes about the current user
+
+### View
+
+The `view` meta helps developers identify the view where a specific signal is coming from. Sometimes apps do not rely on
```

Comment:
Wdyt about something like this as the first sentence?
The `view` meta let's developers define a view to associate signals occured within that view to it. This makes it easy to track specific sections in the UI which may dynamically change without any route changes.

Reply:
Purrrfect! Thanks!